gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,11 +1,5 @@
-#
```

Comment:
should this notice be kept? not a part of `Datadog` code anyway

Reply:
It was added by the automatic tool in IntelliJ but yeah, no point in having this notice here, it's mostly a configuration file